magnetis/astro-native
# feat(icons): add icons

# What

Add icons

# Why

Improve our icons base sheet

# How

- Add `QuestionMark`, `Star` and `StarFilled` icons

# Sample

https://user-images.githubusercontent.com/13890272/142223327-bd878492-b095-45ff-9241-e7d34a88f647.mov

# QA

Run the project locally, open storybook, select the icons and confirm added icons
